provider/cmd/run.go

```diff
@@ -104,6 +105,13 @@ func RunCmd() *cobra.Command {
 		Use:          "run",
 		Short:        "run akash provider",
 		SilenceUsage: true,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			leaseFundsMonInterval := viper.GetDuration(FlagLeaseFundsMonitorInterval)
+			if leaseFundsMonInterval > 0 && leaseFundsMonInterval < time.Minute {
+				return fmt.Errorf("\"lease-funds-monitor-interval\" contains invalid value. expected 0 or >=1m")
```

Comment:
Triple quotes here should remove the need for escapes, and you can use `errors.New` since it doesn't look like you are formatting anything.